jsonschema/validators.py

```diff
@@ -356,7 +356,7 @@ def resolve_fragment(self, document, fragment):
 
         """
 
-        fragment = fragment.lstrip(u"/")
+        fragment = fragment.lstrip(u"#").lstrip(u"/")
```

Comment:
Would you want functionality, perhaps in another method, to convert a pointer into a fragment, by stripping the `#`, etc, or do you want to close the request?